packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -98,4 +99,14 @@ export class StartSiteFlow {
 		const elementHandle = await this.page.waitForSelector( selectors.themePreviewIframe );
 		return ( await elementHandle.contentFrame() ) as Frame;
 	}
+
+	/**
+	 *  Get the current category from theme selection screen.
+	 *
+	 *  @returns Current category name
+	 */
+	async getCurrentCategory(): Promise< string > {
+		const locator = this.page.locator( `${ selectors.categoryItem }[aria-pressed=true]` );
```

Comment:
That's a good point. Since I will be referring to the selector anyway, it's easier to maintain if I add it directly. Nice catch!

Reply:
> I try to avoid concatenating an additional selector to the definition

Dan has a point about this, but I don't always follow this soft rule either. Things are still pretty fluid at Automattic and our best practices evolve from month to month.

Optional suggestion: For this particular selector, one way to make it more versatile would be to follow the example set in another selector above:

```
individualThemeContainer: ( name: string ) => `.design-button-container:has-text("${ name }")`
```

Instead of accepting `name` it could accept a boolean to be passed into the `aria-pressed=<boolean>`, thus consolidating the selector into one.